landing_page/static/landing_page/js/movie.js

```diff
@@ -0,0 +1,44 @@
+
+// Buttons for the movie page's 
+// context change depending on button clicked, testing visuals
+function movieOverview(){
+    document.getElementById("movie_summary").innerHTML = 
+    `Description of <b>Dark Knight</b> Movie: Movie Summary/Plot
+    `;
+}
+function movieDetails(){
+    document.getElementById("movie_summary").innerHTML = 
+    `<div>Director(s):
+    </div>
+    <div> 
+    Writer(s):
+    </div>
+    <div>
+    Star(s):
+    </div>
+    <div>
+    Genre(s):
+    </div>
+    `;
+}
+// movie available to watch 
+function movieWatching(){
+    document.getElementById("movie_summary").innerHTML = 
+    `<div>Streaming: 
+    </div>
+    <div>Buy: 
```

Comment:
Currently we only have the streaming availability saved but it is possible for us to obtain the Buy/Rent options as well